Sources/MapboxMaps/Snapshot/Snapshotter.swift

```diff
@@ -251,6 +257,14 @@ public class Snapshotter {
                                 bearing: bearing?.NSNumber,
                                 pitch: pitch?.NSNumber))
     }
+
+    // MARK: - Telemetry
+    internal func sendTurnstileEvent() {
+        let accessToken = ResourceOptionsManager.default.defaultAccessToken()
+        let eventsManager = EventsManager(accessToken: accessToken)
+
+        eventsManager.telemetry.turnstile()
```

Comment:
Let's avoid accessing the `telemetry` property of `EventsManager`. Instead, we should create some internal API on `EventsManager` that provides an abstraction. That will hopefully limit the scope of changes when we adopt core telemetry.